snuba/datasets/configuration/issues/entities/search_issues.yaml

```diff
@@ -0,0 +1,101 @@
+version: v1
+kind: entity
+name: search_issues
+
+schema:
+  [
+    { name: organization_id, type: UInt, args: { size: 64 } },
+    { name: project_id, type: UInt, args: { size: 64 } },
+    { name: group_id, type: UInt, args: { size: 64 } },
```

Comment:
No primary hash? Just group_id and fingerprints?

Reply:
What do we actually use `primary_hash` for in snuba? I don't think it's really used much in any search functionality.

If we want to include it, then we can extract it out of `fingerprints[0]` or make sure we pass it via eventstream.

Reply:
👍 will add it now.

Reply:
Can you clarify the relationship between the occurrence_id and the primary hash / group id?

Reply:
An event can have multiple occurrences in it. An occurrence currently just has one fingerprint, although we left this as a list for use in the future. Each occurrence represents an issue happening one time.

So in terms of how they relate:
 - An occurrence will have a primary hash that is just the single fingerprint. These will never change
 - An occurrence is related to a group id in the same way an error event is related to a group. So in the future, we'd support merging/deletions and use replacements.